insights/parsers/sos_conf.py

```diff
@@ -0,0 +1,43 @@
+"""
+SosConf - file ``/etc/sos.conf`` or ``/etc/sos/sos.conf``
+=========================================================
+"""
+from insights.core import IniConfigFile
+from insights.core.filters import add_filter
+from insights.core.plugins import parser
+from insights.specs import Specs
+
+add_filter(Specs.sos_conf, "[")
```

Comment:
TBH, it's not recommended to add the filters of the spec here in the parser, especially for a spec that is not so common as the `ps aux/ps auxww`. How about moving the filters to the rule's side?

Reply:
@xiangce yes, works for me. I'll update it.